plugin/kubernetes/handler_test.go

```diff
@@ -525,7 +524,7 @@ func (APIConnServeTest) Run()                                      {}
 func (APIConnServeTest) Stop() error                               { return nil }
 func (APIConnServeTest) EpIndexReverse(string) []*object.Endpoints { return nil }
 func (APIConnServeTest) SvcIndexReverse(string) []*object.Service  { return nil }
-func (APIConnServeTest) Modified() int64                           { return time.Now().Unix() }
+func (APIConnServeTest) Modified() int64                           { return 2 }
```

Comment:
What's the specific reason for this change?

Reply:
I don't recall for certain. I suspect so that the serial is static during the tests, not changing based on when the test is run, but I don't see a change that adds a Serial check, so it could be extraneous.

Reply:
Hmm ... I suspect it's part of the notify implementation, (which I had not posted because it depends on code that wasnt merged yet: #3420). 
Last November, I had Transfer+Notify working for both kubernetes and k8s_external. This may have snuck in from that.

Reply:
Generally would say nit, which shouldn't be in the PR. Doesn't break or add anything so also ok to keep it.

Reply:
Actually - it may be directly related to this change.  There are tests here for incremental transfers, which check the serial.   Making the initial serial a low static number may have been to induce a "serial not changed" scenario to verify the IXFR short response when the zone is not changed.

But looking at the IXFR tests now, I don't see how they are actually working. On the surface, they look invalid. Still re-catching my bearings.

Sorry, it's been so long I don't have a crisp memory of the work.

Reply:
OK, ... I see, the IXFR tests _are_ valid, and the "2" is there for the purpose of testing the IXFR fallback vs empty incremental response.  So it _is_ directly related to this change.

Reply:
I'll add a comment to the code to explain it.

Reply:
Ahh now it makes sense. Thanks for bringing me up to speed.

Was missing it being used as the empty initialization and then check against hardcoded serials via k.Transfer().

All good.